<ui version="4.0" >
 <class>GfxCentralClass</class>
 <widget class="QWidget" name="GfxCentralClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>gfxcentral</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" native="1" name="viewFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" native="1" name="trackViewC" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="miniTrack" >
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QWidget" native="1" name="widget_4" >
         <property name="geometry" >
          <rect>
           <x>220</x>
           <y>0</y>
           <width>131</width>
           <height>18</height>
          </rect>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="mini_goPrev" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="mini_drag" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="mini_goNext" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2" >
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toggleTrackEdit" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>://track/toggle_track.png</normaloff>://track/toggle_track.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TimeLine" name="graphicsView" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>233</green>
                 <blue>216</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>233</green>
                 <blue>216</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>236</red>
                 <green>233</green>
                 <blue>216</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground" >
            <bool>true</bool>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>639</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>232</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="setKey" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Auto Key</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>://track/set_key.png</normaloff>://track/set_key.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" native="1" name="widget_3" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="autoKeyMode" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Auto Key</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="setKeyMode" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Set Key</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" native="1" name="widget_2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="goFirst" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/go_first.png</normaloff>://track/go_first.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>3</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="goPrev" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/go_prev_frame.png</normaloff>://track/go_prev_frame.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="play" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/play.png</normaloff>://track/play.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="goNext" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/go_next_frame.png</normaloff>://track/go_next_frame.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_2" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>3</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="goLast" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/go_last.png</normaloff>://track/go_last.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="keyMoveMode" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/key_move_mode.png</normaloff>://track/key_move_mode.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_3" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>3</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="curTime" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>18</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="timeConfig" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset>
                  <normaloff>://track/time_config.png</normaloff>://track/time_config.png</iconset>
                </property>
                <property name="autoRaise" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>TimeLine</class>
   <extends>QGraphicsView</extends>
   <header>timeline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/editor.qrc" />
 </resources>
 <connections/>
</ui>
